app/controllers/RejectedNotificationsController.scala

```diff
@@ -48,7 +48,7 @@ class RejectedNotificationsController @Inject() (
     extends FrontendController(mcc) with I18nSupport with Logging {
 
   def displayPage(id: String): Action[AnyContent] = (authenticate andThen verifyEmail).async { implicit request =>
-    if (newErrorReportConfig.isNewErrorReportEnabled)
+    if (newErrorReportConfig.isNewErrorReportEnabled && getValue(errorReportView).getOrElse("default") != "original")
```

Comment:
Maybe use "version1|version2" rather than "default|original", more easily understood. Also should create some constants for these values rather than repeating these strings :wink: